setup.py

```diff
@@ -17,11 +17,16 @@
           'Programming Language :: Python :: 3 :: Only'
       ],
       install_requires=[
-          'singer-python==5.8.1',
+          'pipelinewise-singer-python==1.*',
```

Comment:
Is there any danger in using a wildcard for a dependency ?

Reply:
None, pip will always install `pipelinewise-singer-python` version where the Major is 1, that way if we add bug fixes (patch is bumped) or new features(minor is bumped ) to `pipelinewise-singer-python`, `tap-postgres` will always include those because we only fixed the Major.

Basically, we wouldn't have to update this requirement file unless we introduced breaking changes in `pipelinewise-singer-python`, i.e bumped the major, we just have to reinstall `pipelinewise-tap-postgres`.